cypress/e2e/indexCards.cy.ts

```diff
@@ -1,5 +1,5 @@
-/// <reference types="cypress" />
 /// <reference types="cypress-real-events" />
+/// <reference types="../support" />
```

Comment:
It took me a while to realize the `support/index.ts` file the Cypress guides were referring to is actually an `e2e.ts` or `component.ts` depending on what type of testing you're doing. By properly pointing at the `support` folder and using the TS migration guide, you get an easy way to add custom commands and types.